src/processing/calibrate.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import cv2
+import yaml
+
+def calibrate():
+    """
+    Calibrates both cameras for further processes if necessary.
+
+    Returns:
+        mtx: Camera matrix for restoration
+        dist: Distortion coefficient for restoration
+    """
+    
+    """Camera initialization"""
```

Comment:
Source code comments can be made using the regular `# {comment}` syntax. Only the high-level function documentation needs to be in docstring format.

Reply:
Ah i see